easybuild/easyconfigs/r/Racon/Racon-1.4.7-fosscuda-2019b.eb

```diff
@@ -0,0 +1,163 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'CMakeMake'
+
+name = 'Racon'
+version = '1.4.7'
+
+homepage = 'https://github.com/clara-genomics/racon-gpu'
+description = "Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/clara-genomics/racon-gpu/archive'],
+        'filename': '%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/clara-genomics/ClaraGenomicsAnalysis/archive'],
+        'filename': 'ClaraGenomicsAnalysis.zip',
```

Comment:
We should use a datestamp here, in case we later use more recent commits in future updates.

We usually use the datestamp of the commit, so something like `ClaraGenomicsAnalysis-20190814.tar.gz` in this caase

Reply:
Those commits are directly taken from .gitmodules locked versions for targeted release.
In case it is needed to update those, we should use the locked versions for specific release.
It is way easier to obtain commit hash (copy from URL), than translate it to date at which it was issued (finding the date of the commit somewhere in the github page), when updating the easyblock.

Reply:
Oh, the date was for filename and not download. I get it now :)

Reply:
Yes, indeed. A commit ID is meaningless to humans, a datestamp has meaning and sorts nicely.